setup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
Out of curiosity : where is the setup.py file used in this PR? Is it implicitly called by the 

```
    build:
      context: ./WAM
```
command of the `docker-compose.yml` file?

Reply:
It's used by WAM-Apps to install WAM as package under RTD
See https://github.com/rl-institut/WAM_APP_stemp_mv/blob/doc/doc/requirements.txt

Reply:
Is this resolved?

Reply:
Needs no resolving - just a question